doc/source/io.rst

```diff
@@ -214,8 +214,20 @@ na_values : scalar, str, list-like, or dict, default ``None``
   for a list of the values interpreted as NaN by default.
 
 keep_default_na : boolean, default ``True``
-  If na_values are specified and keep_default_na is ``False`` the default NaN
-  values are overridden, otherwise they're appended to.
+  Whether or not to include the default NaN values when parsing the data.
+  Provided that `na_filter` is True, depending on whether `na_values` is
```

Comment:
the 2nd sentence is slightly confusing here

Reply:
Fixed.